lib/langchain/llm/aws_bedrock.rb

```diff
@@ -61,7 +61,7 @@ def initialize(completion_model: DEFAULTS[:completion_model_name], embedding_mod
         .merge(embedding_model_name: embedding_model)
 
       chat_parameters.update(
-        model: {default: @defaults[:chat_completion_model_name]},
+        model: {default: @defaults[:completion_model_name]},
```

Comment:
I think we actually need to add `chat_completion_model_name` key and value to the hash above:
```ruby
    DEFAULTS = {
      completion_model_name: "anthropic.claude-v2",
      embedding_model_name: "amazon.titan-embed-text-v1",
```

Reply:
Looks like this 🤔 
![CleanShot 2024-10-02 at 10 02 02](https://github.com/user-attachments/assets/5abe7869-7be8-408f-9b80-b46da284e6ca)

Reply:
BTW, we are using the same `completion_provider` to compose parameters & response for both `chat` & `complete`. So I think we can use the same `completion_model_name` for both as well. WDYT?